src/main/java/com/faforever/moderatorclient/api/FafApiCommunicationService.java

```diff
@@ -84,6 +83,8 @@ public FafApiCommunicationService(ResourceConverter resourceConverter, Applicati
                 .additionalMessageConverters(jsonApiMessageConverter)
                 .errorHandler(jsonApiErrorHandler)
                 .rootUri(apiBaseUrl);
+        headers = new HttpHeaders();
```

Comment:
@bukajsytlos @Brutus5000  for post i got headers working, however i still need to due it for patch. Do you know if there is a way to set headers in general?